### ISSUE TITLE
Request documentation of tensorboard logs output

### ISSUE THREAD

**Issue by nate-bush** [pytorch/fairseq#2616]
I had a difficult time tracking down and understanding the model training output in the tensorboard logs. Here is what I learned (which I **think** is correct). It would be helpful do add this int he documentation.

loss
- cross entropy loss
- loss

nll_loss
- cross entropy loss
- loss_sum / ntokens

ppl
- perplexity
- average of nll_loss

wps
- words per second
- average number of tokens per second

train_wall
- train wall clock time
- number of seconds to finish one training step. Do forward, backward and parameter update.

wall
- train wall clock time
- includes train_wall, logging, checkpointing model, etc.

loss_scale
- fp16 loss scale
- dynamic loss scale calculated by fp16 loss sclaer

clip
- clipped gradient norm value
- 100 if gradient norm is greater than clip norm value argument, otherwise 0

gnorm
- norm of the gradient

lr
- learning rate

ups
- updates per second
- time between logging updates

wpb
- words per batch
- average number of words per batch

bsz
- number of sentences per batch

num_updates
- learning rate schedule
- update the learning rate after this many steps

**Comment by mdoumbouya** [comment]
This should be re-opened. The [official documentation](https://fairseq.readthedocs.io/en/v0.10.2/search.html?q=tensorboard&check_keywords=yes&area=default#) still doesn't include information about the variables that are logged in tensorboard.
